common/models/load_balancer_summary.go

```diff
@@ -0,0 +1,8 @@
+package models
+
+type LoadBalancerSummary struct {
```

Comment:
I guess, to answer my own question, there are benefits in clarity throughout the code. That's one good reason. Another would be not having to pass around non-critical data relating to List operations, although I wonder how much performance / memory gains we really get from that...